src/runner.ts

```diff
@@ -34,8 +35,18 @@ export const runner = async () => {
       );
     }
 
+    log('Creating shot folders');
     createShotsFolders();
+
+    log('Creating shots');
     const shotItems = await createShots();
+
+    if (shotItems.length === 0) {
+      log(`Exiting process with nothing to compare.`);
+      process.exit(1);
+    }
+
+    log('Checking differences');
```

Comment:
little bit more logging so we are following the process in a more pinpointed fashion